### ISSUE TITLE
Tiles have black borders at the top, varying by zoom level

### ISSUE THREAD

**Issue by indig0fox** [gruppe-adler/grad_mtg#9]
Hello! I attempted to use this tool to export Altis, zoom levels 2-5. However, the black bar at the top of the tilesets varies in overall width and is causing positioning issues with Leaflet (where I'm trying to import them). It shifts the entire resulting tileset and creates unpredictable location changes of the terrain.

I was hoping to get high-resolution tiles for use in an AAR system.

The functionality of mirroring the in-game sat map is ideal for my purposes and I was wondering if this could be easily remedied?

Zoom 4 
![image](https://user-images.githubusercontent.com/23415303/134621441-23df9b9b-c1f0-4db2-af10-7226fd3605dd.png)
Zoom 5
![image](https://user-images.githubusercontent.com/23415303/134621457-f12c217d-e457-453b-9020-7f890fc95de5.png)

**Comment by DerZade** [comment]
Hello!

First of all, this project is abandoned in favour of  [grad_meh](https://github.com/gruppe-adler/grad_meh). I don't know why I haven't archived it yet, but I'll look into it next week.

If you still want to use it, you should check out [this implementation of a custom Leaflet-Layer](https://github.com/gruppe-adler/maps.gruppe-adler.de/blob/714a3b7feb9363ec3b250a35ba6e6cf4bc5601f4/frontend/src/models/ArmaTileLayer.ts). It uses `crs.Simple`, which you can read more about [here](https://leafletjs.com/examples/crs-simple/crs-simple.html).

**Comment by indig0fox** [comment]
Our project actually already uses CRS.Simple. A legacy thing that I've found a way of replacing but, thus it is, which makes this the most helpful comment I've ever seen on Github! Thank you for the sample.

I like the tiling of grad_meh, but many satellite images are too small for our purposes and my images seem to come out only half-rendered. See attached. But that's an issue for if MTG doesn't work.

I appreciate you mentioning this isn't maintained any longer, I'll bear that in mind as/if grad_meh grows further.
![2](https://user-images.githubusercontent.com/23415303/134757003-4cef3036-6c58-4142-bfb5-2400e12af798.png)

**Comment by DerZade** [comment]
I'm glad my custom tile layer helped.

If I understand correctly, you are still having problems?
Could you please elaborate on that! I don't really understand what you are trying to say 😅.

**Comment by indig0fox** [comment]
I looked at the tiles from grad_meh, and realized that they're just really big and aren't broken down into different zoom level subdirectories (which I updated my tile source URL to account for). Just have to account for the new bounds size. But! Will take that to the meh repo.

**Comment by DerZade** [comment]
Yeah `grad_meh` has a fundamentally different goal then `grad_mtg`:
> The aim of this project is not to generate finished map tiles for a WMTS, but to export all data that might be needed to generate such tiles or basically anything else. So the goal is to export everyting interesting about an Arma 3 map. What you do with that data is up to you. The use cases are only limited by your imagination (ﾉ◕ヮ◕)ﾉ*:・ﾟ✧

_(From the `grad_meh` README)_
